frontend/src/views/web/change-password.html

```diff
@@ -4,33 +4,42 @@
             <div class=" ev-card-panel ev-z-depth-5">
                 <div class="ev-md-container ev-panel-title"><strong>Change Password</strong></div>
                 <div class="ev-md-container ev-card-body change-passowrd-card">
-                    <form ng-submit="changepwd.changePassword()">
+                    <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
 
                         <div class="input-field align-left">
-                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" focus-if>
-                            <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
-                            <label for="old_password" data-error="wrong" data-success="right">Old Password*</label>
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.old_password">{{changepwd.wrnMsg.old_password}}</div>
+                            <input id="old_password" name="old_password" type="password" text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password"  ng-minlength="8" required focus-if>
+                            <span class="form-icon form-icon-dark"><i class="fa fa-key"></i></span>
+                            <label for="old_password">Old Password*</label>
+                            <div class="wrn-msg text-highlight"  ng-messages="changepwdForm.old_password.$error"  ng-if="changepwdForm.old_password.$touched || changepwdForm.$submitted">{{changepwd.wrnMsg.old_password}}
```

Comment:
remove `{{changepwd.wrnMsg.old_password}}`. No need of this.
Add `<p ng-message="minlength">password is less than 8 characters.</p>`